newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardAppearanceDialog.js

```diff
@@ -134,6 +150,25 @@ function LeaderboardAppearanceDialog({
       setScoreTitleError(i18n._(t`Title cannot be empty.`));
       return;
     }
+    if (
+      isNaN(defaultDisplayedEntriesNumber) ||
+      !Number.isInteger(defaultDisplayedEntriesNumber)
```

Comment:
maybe create a tiny helper `isValidEntry()` that does those 2 checks (and re-use below)